src/test/librados/io_cxx.cc

```diff
@@ -462,6 +462,19 @@ TEST_F(LibRadosIoPP, XattrListPP) {
   }
 }
 
+TEST_F(LibRadosIoPP, CrcZeroWrite) {
+  char buf[128];
```

Comment:
crimson lacks the fix, so I expect this test to fail there. Might we worth adding at a comment at least.

Reply:
It's a simple fix -- can we just go ahead and fix it in crimson rather than letting the test fail?

Reply:
actually, i tested it on crimson and it pass

Reply:
This test fails sporadically with ENOTSUP:

```
/mnt/data/workspace/ceph_linux/src/test/librados/io_cxx.cc:888: Failure
Expected equality of these values:
  0
  ioctx.write("foo", bl, 0, sizeof(buf))
    Which is: -95
[  FAILED  ] LibRadosIoECPP.CrcZeroWrite (767 ms)
```

Reply:
I'm thinking about skipping it on Windows for the time being: https://github.com/ceph/ceph-win32-tests/pull/19